### PR TITLE
JENKINS-61094 Reformatting Jenkins plugin source code

### PR DESCRIPTION
**Pull request by sbuhary**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/veracode/DynamicAnalysisResultsAction.java [hunk 1]
```diff
@@ -36,15 +35,16 @@ public DynamicAnalysisResultsAction() {
...
+                    "Missing required information to create a DynamicAnalysisResultsAction.");
         }
         this.scanHistory = scanHistory;
         build = null;
     }
 
     /**
-     * Use by Jenkins framework to display our logo on the left panel on
-     * on the build page
+     * Use by Jenkins framework to display our logo on the left panel on on the
```
Comment (dennisgu1) [inline src/main/java/io/jenkins/plugins/veracode/DynamicAnalysisResultsAction.java]: extra "on"?

File: src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java [hunk 2]
```diff
@@ -67,866 +66,969 @@
...
+
+                    remoteVeracodeFilePath.mkdirs();
+                }
+
+                FilePath[] files = remoteVeracodeFilePath.list(Constant.inclusive);
+
+                // copy the jar if it does not exist
+                if (files.length == 0)
+
+                {
```
Comment (kkyiu) [inline src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java]: Should this be reformat as well.

File: src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java [hunk 3]
```diff
@@ -67,866 +66,969 @@
...
+                            bRet = true;
+                    }
+                }
+            } catch (Exception ex) {
+                ex.printStackTrace();
+                if (getDescriptor().getFailbuild()) {
+                    ps.print(ex.getMessage());
+                }
+            }
+        } else
```
Comment (kkyiu) [inline src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java]: need surrounding braces for the else clause?

File: src/main/java/io/jenkins/plugins/veracode/VeracodePipelineRecorder.java [hunk 4]
```diff
@@ -554,15 +611,18 @@ private boolean runScanFromRemote(Run<?, ?> run, FilePath workspace, TaskListene
...
-                    if (iPosPassword == i) masks[i]=true;
+                    if (iPosPassword == i)
+                        masks[i] = true;
                 } else if (iPosKey != -1) {
-                    if (iPosKey == i) masks[i]=true;
+                    if (iPosKey == i)
+                        masks[i] = true;
                 } else if (iPosProxyPassword != -1) {
-                    if (iPosProxyPassword == i) masks[i]=true;
+                    if (iPosProxyPassword == i)
```
Comment (kkyiu) [inline src/main/java/io/jenkins/plugins/veracode/VeracodePipelineRecorder.java]: need surrounding braces around the if clause?

File: src/main/java/io/jenkins/plugins/veracode/DynamicAnalysisResultsAction.java (hunk 1 above)
Comment (sbuhary) [reply src/main/java/io/jenkins/plugins/veracode/DynamicAnalysisResultsAction.java]: Extra "on" is removed.

File: src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java (hunk 2 above)
Comment (sbuhary) [reply src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java]: Yes, it should be. Done.

File: src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java (hunk 3 above)
Comment (sbuhary) [reply src/main/java/io/jenkins/plugins/veracode/VeracodeNotifier.java]: Added the surrounding braces.

File: src/main/java/io/jenkins/plugins/veracode/VeracodePipelineRecorder.java (hunk 4 above)
Comment (sbuhary) [reply src/main/java/io/jenkins/plugins/veracode/VeracodePipelineRecorder.java]: Added the surrounding braces.

**Pull request merged**